drivers/ethernet/Kconfig.w5500

```diff
@@ -0,0 +1,37 @@
+# W5500 Ethernet driver configuration options
+
+# Copyright (c) 2020 Linumiz
+# Author: Parthiban Nallathambi <parthiban@linumiz.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ETH_W5500
+	bool "ENC28J60C Ethernet Controller"
```

Comment:
You probably meant to say `W5500 Ethernet Controller` here

Reply:
Thanks, that was copy paste typo :) Fixed it